pkg/operator/staticpod/controller/backingresource/backing_resource_controller.go

```diff
@@ -36,7 +36,7 @@ const (
 // (templated with the config) if they differ.
 type BackingResourceController struct {
 	targetNamespace      string
-	operatorConfigClient v1helpers.StaticPodOperatorClient
+	operatorConfigClient v1helpers.OperatorClient
```

Comment:
did you try this?  I suspect it will stop out the node statuses (mistake @sttts and I realized recently)